app/templates/services/previous_services.html

```diff
@@ -0,0 +1,89 @@
+{% extends "_base_page.html" %}
+{% import "toolkit/summary-table.html" as summary %}
+{% import "macros/submission.html" as submission %}
+
+{% block page_title %}Copy your {{ framework.name }} services – Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with items = [
+      {
+        "link": "/",
+        "label": "Digital Marketplace",
+      },
+      {
+        "link": url_for(".dashboard"),
+        "label": "Your account",
+      },
+      {
+        "link": url_for(".framework_dashboard", framework_slug=framework.slug),
+        "label": "Apply to " + framework.name,
+      },
+      {
+        "link": url_for(".framework_submission_lots", framework_slug=framework.slug),
+        "label": "Services",
+      },
+      {
+        "link": url_for(".framework_submission_services", framework_slug=framework.slug, lot_slug=lot.slug),
+        "label": lot.name,
+      }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+
+  {% include "partials/service_warning.html" %}
+
+  {% if copy_all %}
+    <form action="{{ url_for('.copy_all_previous_services', framework_slug=framework.slug, lot_slug=lot.slug) }}" method="POST">
+      <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
+      {%
+        with
+        message = "Are you sure you want to add all your {} services?".format(lot.name|lower),
+        type = "destructive",
```

Comment:
I think this should be a green flash message. I recognise that it's a strong action to take, but it's definitely not destructive and I think red sends the wrong signal.

If we have some kind of 'warning' type (orange maybe), that would be fine. But I think we want to keep destructive/red messages to things that actually do deletions.

Reply:
Constance and I spend a bit of time looking at this. We decided that whilst not destructive, the implications of copying all your services when you didn't mean to and then having to delete them all from your list of drafts, merited a red message.

There _is_ and orange box, but then it looks really weird with the red button. If Constance is about, have a chat with her about it and see what she thinks.

Reply:
Copying services is a positive thing. It's going to help the supplier. I think it feels like we're warning them against doing it and that's going to put the supplier in a weird place mentally.

It's a big action, and that's why we're making the user confirm they want to take it. But it's not a _bad_ action.

I'll chat to Constance if I see her.

Reply:
P.S. I probably shouldn't have mentioned the orange box as I think @laurenceberry wanted us to get rid of it (and maybe we already did).